rebasehelper/utils.py

```diff
@@ -867,9 +867,12 @@ def session_maker(cls, baseurl=None):
             return koji_session
         try:
             koji_session.krb_login()
-        except koji.krbV.Krb5Error:
-            # fall back to login using certificate
-            koji_session.ssl_login(cls.cert, cls.ca_cert, cls.ca_cert)
+        except koji.krbV.Krb5Error as e:
+            try:
+                # fall back to login using certificate
+                koji_session.ssl_login(cls.cert, cls.ca_cert, cls.ca_cert)
+            except Exception:
+                raise RebaseHelperError('Kerberos login failed. The reason is: {}' % (six.text_type(e)))
```

Comment:
This is not correct either. It discards the error raised from `koji_session.ssl_login()`. Proper way to handle this would be to first try to login using kerberos, print an error if it fails, fall back to login using certificate and print an error and stop execution if that fails either.
But as I wrote in my previous comment, I think logging in using certificate can be dropped, because it doesn't work anymore (actually since [2016-12-12](https://fedoraproject.org/wiki/Infrastructure/Kerberos)).